site/site.go

```diff
@@ -672,3 +720,67 @@ func RenderStaticErrorPage(rw http.ResponseWriter, r *http.Request, data ErrorPa
 		return
 	}
 }
+
+type binHashCache struct {
+	binFS http.FileSystem
+
+	hashes map[string]string
+	mut    *sync.RWMutex
+	sf     *singleflight.Group
```

Comment:
```suggestion
	mut    sync.RWMutex
	sf     singleflight.Group
```

Since we're using pointer receivers, it's preferable to have useful zero values. This avoids having to assign them in new.